module/platform/NUSense/HardwareIO/src/HardwareIO.cpp

```diff
@@ -85,11 +86,74 @@ namespace module::platform::NUSense {
             // Tell the stream reactor to connect to the device
             emit(std::make_unique<ConnectSerial>(device, baud));
 
+            // TODO: Apply the servo offsets within NUSense instead
```

Comment:
so this PR just sends the information right, it doesn't actually have NUSense using it yet?

Reply:
hence its a draft atm. Will put it up when ive got the code for the firmware